src/rules/no_external_imports.rs

```diff
@@ -55,10 +55,11 @@ struct NoExternalImportHandler;
 impl NoExternalImportHandler {
   fn check_import_path(&self, decl: &ImportDecl, ctx: &mut Context) {
     let parsed_src = ModuleSpecifier::parse(decl.src.value());
-    let maybe_file_path = ctx.specifier().to_file_path().ok();
-    let file_stem = maybe_file_path
-      .as_ref()
-      .and_then(|p| p.file_stem())
```

Comment:
for reviewers: Url's `to_file_path` method isn't provided for wasm32-unknown-unknown. As a workaround, we extract the last segment from specifier's path part, and treat it as a file name.